Pipfile

```diff
@@ -13,16 +13,14 @@ numpy = "*"
 psychrolib = "*"
 pythermalcomfort = "*"
 icecream = "*"
-pydantic = ">=2.8.2
-plotly = ">=5.23.0"
-scipy = ">=1.14.0
+pydantic = "*"
```

Comment:
your codebase was not up to date, you should pull the remove changes before doing a pull request since someone else has created a pull request before you.

Reply:
Do I need close the pull request and pull the remove changes then pull the request?